binaries/setup.py

```diff
@@ -41,5 +41,10 @@
   package_data={
       # Only add the corresponding platform specific binaries to the wheel.
       streamer_binaries.__name__: platform_binaries,
-  }
+  },
+  install_requires=[
+      # This is only used for Linux, and only supports Linux, but can be
+      # installed on all platforms and won't crash if imported or run.
+      'distro',
+  ],
```

Comment:
can't we do something like this: https://github.com/shaka-project/shaka-streamer/blob/09df9fbbef53fdf8430cd898ea671bf81f9fd125/setup.py#L34

so we don't have to install it on other platforms.

Reply:
Done